examples/specs/box_plot_minmax_full.vl.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "A box plot showing median, min, and max in the US population distribution of age groups in 2000.",
+  "data": {"url": "data/population.json"},
+  "layer": [
+        {
+          "mark": "rule",
+          "encoding": {
+            "x": {"field": "age","type": "ordinal"},
+            "y": {
+              "aggregate": "min",
+              "field": "people",
+              "type": "quantitative",
+              "axis": {"title": "population"}
+            },
+            "y2": {
+              "aggregate": "max",
+              "field": "people",
+              "type": "quantitative"
+            },
+            "size": {"value": 5},
+            "color": {"value" : "white"}
```

Comment:
Just to make sure, only the bar marks should have the color property in their encoding?